python_modules/dagster/dagster/_core/definitions/decorators/sensor_decorator.py

```diff
@@ -214,7 +214,8 @@ def multi_asset_sensor(
 
     Args:
         asset_keys (Optional[Sequence[AssetKey]]): The asset keys this sensor monitors. If not
-            provided, asset_selection argument must be provided.
+            provided, asset_selection argument must be provided. To monitor SourceAssets, you must
```

Comment:
@jamiedemaria sorry I had been meaning to comment on this. I think it would be more accurate to say "to monitor assets that aren't defined in the repo that this sensor is part of" or something like that. Not all of these will be SourceAssets.  I also think we should clarify that asset_selection only works within the same repo.